impressionist.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "friendly_id"
   s.add_dependency 'nokogiri', RUBY_VERSION < '2.1.0' ? '~> 1.6.0' : '~> 1'
-  s.add_dependency 'rails', '>= 3.2.15'
+  s.add_dependency 'rails', '>= 3.2.15', '< 7.1'
```

Comment:
Without this constraint, we were getting an `ArgumentError` when creating a Rails 7.1 app.